packages/eslint-plugin/src/rules/consistent-type-imports.ts

```diff
@@ -50,7 +54,8 @@ type MessageIds =
   | 'valueOverType'
   | 'noImportTypeAnnotations'
   | 'someImportsInDecoMeta'
-  | 'aImportInDecoMeta';
+  | 'aImportInDecoMeta'
+  | 'inlineTypes';
```

Comment:
The new `inlineTypes` message ID is no longer used anywhere.

```suggestion
  | 'aImportInDecoMeta';
```